etc/parameters/default.json

```diff
@@ -1042,7 +1042,9 @@
     }
   },
   "penalty_shot_direction_estimation": {
-    "moving_distance_threshold": 0.2
+    "moving_distance_threshold": 0.2,
+    "minimum_velocity": -0.5,
```

Comment:
This disables the velocity check, right?

Reply:
We dont have a velocity check in the main right now, this creates one